README.md

```diff
@@ -224,14 +226,21 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
 | `AUTH`   | used to send authentication challenges                  | [42](42.md) |
 | `COUNT`  | used to send requested event counts to clients          | [45](45.md) |
 
-## Standardized Tags
+## Tags
```

Comment:
```suggestion
## Tags

Different kinds may use the same tag differently.
```

Reply:
How about adding it to NIP-01?

Reply:
These explanations, duplicating them?
Doesn't sound like a good idea to me.

Reply:
No, they will not be explained here.